lib/views/game.html.erb

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Codebreaker</title>
+    <link rel="stylesheet" href="/stylesheets/application.css" type="text/css">
+  </head>
+  <body>
+    <div id="container">
+      <h1>Codebreaker!</h1>
+      <h2>Name: '<%= data_session(:name) %>'<h2>
+      <h3>Attempts: '<%= game.attempt_count %>'<h3>
+      <p><%= data_session(:hint) ? "Hint: #{data_session(:hint)}" : 'Hint: *' %></p>
+      <a href="/hints"><button>Hint!</button></a>
+      <% if game.game_over? == false %>
```

Comment:
better to use templates for those cases
check game state out of the view and render the template that needed